doc/model/fluid.rst

```diff
@@ -0,0 +1,176 @@
+=====
+Fluid
+=====
+
+.. note::
+
+   This model document has been updated to be *MIRGE-Com* specific, and to include multi-component mixtures
+   with chemical reactions.
+
+.. important::
+
+   *MIRGE-Com* main repo branch currently provides fluid simulation operators and utilities for inviscid
+   flow (i.e. the :ref:`Euler equations<Euler-eqns>`).  Viscous flow simulation capabilities are forthcoming.
+
+.. raw:: latex
+
+    \def\RE{\operatorname{RE}}
+    \def\PR{\operatorname{PR}}
+
+.. raw:: html
+
+    \(
+    \def\RE{\operatorname{RE}}
+    \def\PR{\operatorname{PR}}
+    \)
+
+.. _NS-eqns:
+
+*MIRGE-Com* provides capabilities for solving the compressible Navier-Stokes equations for viscous flows and
+the :ref:`Euler equations<Euler-eqns>` equations for inviscid flows of reactive fluid mixtures. *MIRGE-Com*
+supports reactive fluid mixtures with a number of mixture species = $N_s$ on unstructured meshes and
+:ref:`discretizes the equations in a Discontinuous Galerkin setting<disc-strat>`.
+
+The formulation presented here follows from [Ihme_2014]_ and [Cook_2009]_. The governing equations, written in conservative form, are summarized as follows:
```

Comment:
Mention that the docs use the Einstein summation convention.

Reply:
(cc010df)